social_protection/tests/data.py

```diff
@@ -91,3 +91,15 @@
     "date_valid_from": "2023-01-01",
     "date_valid_to": "2023-12-31",
 }
+
+service_group_add_payload = {
+    "status": "Potential",
```

Comment:
I think now status is not 'Potential' but 'POTENTIAL' based on enum value and your previous approaches